cdk/lib/support-workers.ts

```diff
@@ -109,18 +100,67 @@ export class SupportWorkers extends GuStack {
       },
     };
 
-    const createLambda = (
+    const createScalaLambda = (
       lambdaName: string,
       additionalPolicies: PolicyStatement[] = []
     ) => {
       const lambdaId = `${lambdaName}Lambda`;
       const lambda = new GuLambdaFunction(this, lambdaId, {
         ...lambdaDefaultConfig,
+        fileName: `support-workers.jar`,
+        runtime: Runtime.JAVA_21,
         handler: `com.gu.support.workers.lambdas.${lambdaName}::handleRequest`,
         functionName: `${this.stack}-${lambdaName}Lambda-${this.stage}`,
```

Comment:
assume everything is just a duplicate between scala->ts other than these 4 lines?

Reply:
The functionName is the same so it's just the three lines difference

Reply:
consider making it a function that only takes those 3 parameters, to make it clearer they are the same